### PR TITLE
Updated Welcome Page

### PR DESCRIPTION
**Pull request by rmroot**
connects #214 

Updates to the welcome page. Adds ability to go back into latest visits.

### CODE REVIEW

File: src/app/core-components/welcome/welcome.component.html [hunk 1]
```diff
@@ -1,99 +1,174 @@
...
+                                energy assessment details that will allow you to discover additional non-energy
+                                benefits.
                             </p>
+                            <div class="d-flex justify-content-center">
+                                <button class="btn btn-outline-success" (click)="openWizardModal()">
+                                    <fa-icon [icon]="faWandMagicSparkles"></fa-icon>
+                                    Setup Wizard</button>
+                            </div>
+                            <hr>
+                            <div class="list-content">
```
Comment (RLiNREL) [inline src/app/core-components/welcome/welcome.component.html]: I couldn't find the class definition for list-content.

File: src/app/core-components/welcome/welcome.component.ts [hunk 2]
```diff
@@ -33,4 +84,11 @@ export class WelcomeComponent {
   openWizardModal() {
     this.sharedDataService.createAssessmentModalOpen.next(true);
   }
+
+  goToVisit(visit: IdbOnSiteVisit) {
+    this.companyIdbService.setSelectedFromGUID(visit.companyId);
+    this.facilityIdbService.setSelectedFromGUID(visit.facilityId);
+    this.onSiteVisitIdbService.setSelectedFromAssessmentGUID(visit.assessmentIds[0]);
```
Comment (RLiNREL) [inline src/app/core-components/welcome/welcome.component.ts]: Is `visit.assessmentIds[0]` a concern if no assessments have been created for the visit?
Currently, it seems that user is allowed to set up a new visit without creating any assessment

File: src/app/core-components/welcome/welcome.component.html [hunk 3]
```diff
@@ -1,99 +1,174 @@
...
+                                    </a> -->
+                                    <a>
+                                        <fa-icon [icon]="faExternalLink"></fa-icon>
+                                        NEBs Knowledge Library
+                                    </a>
+                                    <p class="card-text fw-light">
+                                        <span class="badge bg-warning">Coming Soon</span>
+                                        A knowledge library of content associated to the study of non-energy
+                                        benefits
+                                        and the process around identifying them has been currated by NREL.
```
Comment (RLiNREL) [inline src/app/core-components/welcome/welcome.component.html]: a typo, "currated" -> "curated"

File: src/app/core-components/welcome/welcome.component.html (hunk 1 above)
Comment (rmroot) [reply src/app/core-components/welcome/welcome.component.html]: removed

File: src/app/core-components/welcome/welcome.component.html (hunk 3 above)
Comment (rmroot) [reply src/app/core-components/welcome/welcome.component.html]: fixed

File: src/app/core-components/welcome/welcome.component.ts (hunk 2 above)
Comment (rmroot) [reply src/app/core-components/welcome/welcome.component.ts]: updated thanks

**Review by RLiNREL — APPROVED** [review]

**Pull request merged**